test/it/fixtures/test.v2.suite.ts

```diff
@@ -82,16 +83,16 @@ class CustomSequence2 {
     @test public step3() { /* should be skipped */ }
 }
 
-@suite(slow(10))
+@suite(slow(SLOW))
 class Slows {
     @test public pass1(done) {
-        setTimeout(done, 1);
+        setTimeout(done, IN_TIME);
     }
     @test public pass2(done) {
-        setTimeout(done, 20);
+        setTimeout(done, SLOW);
```

Comment:
This should probably fail, maybe OVERLY_SLOW?

Reply:
Yes, done that.